engine/runtime/src/main/java/org/enso/interpreter/runtime/type/Constants.java

```diff
@@ -1,22 +1,24 @@
 package org.enso.interpreter.runtime.type;
 
-/** Types defined in the Standard.Builtins module. */
+/** Hard-coded names of builtins. */
```

Comment:
Surely this can be computed dynamically now?

Reply:
That would require us loading stdlib **always**. But could hardcode those in `@BuiltinType`

Reply:
OK, it was possible but required some tricky setup to deal with the order of generation/compilation of Constants